openmls/tests/book_code.rs

```diff
@@ -478,6 +478,21 @@ fn book_operations(ciphersuite: &'static Ciphersuite, backend: &impl OpenMlsCryp
     assert_eq!(members[1].credential().identity(), b"Bob");
     assert_eq!(members[2].credential().identity(), b"Charlie");
 
+    // Check that the `member` and the `members` function are consistent
+    for member in members {
```

Comment:
Can you also check that the number of members is what we expect? Otherwise this could just skip members.